diagnostic.md

```diff
@@ -5,74 +5,80 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+to handle and hold the information so that it can relay it
+to the front end to help interact with the client request.  example would be
+client wants to read a data the backend would gather and send it to front end to
+view
```

Comment:
Good it persists our data so it is stored between sessions. It also allows multiple users to interact over the same app while on different devices.